docs/send-data/index.md

```diff
@@ -7,77 +7,61 @@ tags: [send data, collectors, collection]
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-Learn how to send data to Sumo Logic. In this section, we'll introduce the following concepts:
+Learn how to send data to Sumo Logic, the starting point to using our observability and security solutions. In this section, we'll introduce the following concepts:
 
-<div className="box smallboxwidetop card">
-  <div className="container"><br/>
-  <img src={useBaseUrl('img/icons/data-collection.png')} alt="icon" width="55"/>
-  <h4>Design Your Deployment</h4>
-  <ul>
-  <li><a href="/docs/send-data/choose-collector-source">Choose a Collector and Source</a></li>
-  <li><a href="/docs/send-data/collection">Navigating the Collector and Source UI</a></li>
-  </ul>
-  </div>
-</div>
+:::tip Designing your Deployment
+* [How to choose a Collector and Source](/docs/send-data/choose-collector-source)
+* [How to use the Collection UI](/docs/send-data/collection)
+:::
 
-<br/>
 
 <div className="box-wrapper" markdown="1">
 <div className="box smallbox1 card">
   <div className="container">
-  <img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>Installed Agents</h4>
-  <ul>
-  <li><a href="/docs/send-data/installed-collectors">Installed Collectors</a></li>
-  <li><a href="/docs/send-data/installed-collectors/sources">Explore Installed Collector Sources</a></li>
-  <li><a href="/docs/send-data/sumo-distribution-opentelemetry">OpenTelemetry Distribution</a></li>
-  </ul>
+  <a href="/docs/send-data/sumo-distribution-opentelemetry"><img src={useBaseUrl('img/send-data/otel-color.svg')} alt="icon" width="40"/><h4>Sumo Logic Distro for OpenTelemetry</h4></a>
+  <p>Our OTel distro is a single unified agent to send Logs, Metrics, Traces, and more.</p>
   </div>
 </div>
 <div className="box smallbox2 card">
   <div className="container">
-  <img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>Hosted Collectors</h4>
-  <ul>
-  <li><a href="/docs/send-data/hosted-collectors/configure-hosted-collector">Configuring Hosted Collectors</a></li>
-  <li><a href="/docs/send-data/hosted-collectors">Sources (Amazon, Google, C2C, HTTP, and more)</a></li>
-  </ul>
+  <a href="/docs/send-data/installed-collectors"><img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>Installed Collectors</h4></a>
+  <p>Install a collector in your environment to receive logs and metrics from your Sources.</p>
```

Comment:
```suggestion
  <p>Installed collector is our proprietary agent that you can deploy in your environment to collect logs and some metrics sources.</p>
```

Propose changing the text slightly here, to make sure we higlight that this is also an agent.